Lombiq.Hosting.Tenants.EmailQuotaManagement/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:Lombiq.Hosting.Tenants.EmailQuotaManagement.Services.EmailQuotaService.#ctor(YesSql.ISession,Microsoft.Extensions.Options.IOptions{Lombiq.Hosting.Tenants.EmailQuotaManagement.Models.EmailQuotaOptions},Microsoft.Extensions.Options.IOptions{OrchardCore.Email.DefaultSmtpOptions},Microsoft.Extensions.Options.IOptions{OrchardCore.Email.SmtpOptions},OrchardCore.Modules.IClock,OrchardCore.Security.Services.IRoleService,Microsoft.AspNetCore.Identity.UserManager{OrchardCore.Users.IUser})</Target>
+    <Left>lib/net8.0/Lombiq.Hosting.Tenants.EmailQuotaManagement.dll</Left>
+    <Right>lib/net8.0/Lombiq.Hosting.Tenants.EmailQuotaManagement.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+</Suppressions>
```

Comment:
Since these are auto-generated files that we'll never hand-edit (apart from deleting them), I wouldn't try to keep to the end of file line break practice.